opm/parser/eclipse/EclipseState/Schedule/Well.cpp

```diff
@@ -232,6 +237,47 @@ namespace Opm {
     }
 
 
+
+    void Well::setRFT(size_t time_step, bool value){
+        m_rft->add(time_step, value);
+    }
+
+    bool Well::getRFT(size_t time_step) const{
+        return m_rft->get(time_step);
+    }
+
+    bool Well::getPLT(size_t time_step) const{
+     return m_plt->get(time_step);
+    }
+    void Well::setPLT(size_t time_step, bool value){
+        m_plt->add(time_step, value);
+    }
+
+    int Well::findWellFirstOpen(int startTimeStep) const{
```

Comment:
Not necessary to use `this` in `findWellFirstOpen()` and `setRFTForWellWhenFirstOpen()`